com.valvesoftware.SteamLink.yml

```diff
@@ -0,0 +1,208 @@
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/8540a57312f815f81a45b891b49959d776727fde17579bb6bf1a537996bc9359/qtsvg-everywhere-src-5.14.1.tar.xz
+        sha256: 8540a57312f815f81a45b891b49959d776727fde17579bb6bf1a537996bc9359
+
+  - name: extra-cmake-modules
+    buildsystem: cmake-ninja
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/frameworks/5.78/extra-cmake-modules-5.78.0.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/eede1f0c21f24eca56bd71a805fa76500a8c683bf21982cb33f4bba682d012f9/extra-cmake-modules-5.78.0.tar.xz
+        sha256: eede1f0c21f24eca56bd71a805fa76500a8c683bf21982cb33f4bba682d012f9
+      # also available as:
+      # type: git
+      # url: https://invent.kde.org/frameworks/extra-cmake-modules.git
+      # tag: v5.78.0
+      # commit: 59d4958740a09d43aa5025a1cfe3d7b7d4d725b3
+
+  - name: kdbusaddons
+    buildsystem: cmake-ninja
+    cleanup:
+      - '/bin'
+      - '/include'
+      - '/lib/cmake'
+      - '/lib/mkspecs'
+    config-opts:
+      - -DENABLE_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/frameworks/5.78/kdbusaddons-5.78.0.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/4f91d0b5762243e5e044c087d4faaa584ad5f779fdede138a0048fd4af24d399/kdbusaddons-5.78.0.tar.xz
+        sha256: 4f91d0b5762243e5e044c087d4faaa584ad5f779fdede138a0048fd4af24d399
+      # also available as:
+      # type: git
+      # url: https://invent.kde.org/frameworks/kdbusaddons.git
+      # tag: v5.78.0
+      # commit: e70c696143682bf23509aed4c31d5fc95093d452
+
+      # if we delete the l10n (which we don't want anyway) then it drops the
+      # dependency on QtLinguist
+      - type: shell
+        commands:
+          - rm -fr po
+
+  - name: steamlink-binary
+    sources:
+      - type: archive
+        url: https://repo.steampowered.com/hidden/steamlink/b8b1b94651e7a4cfd6335180697cd35dda771058f942900b5775a69720d218c8/steamlink-1.1.73.179.tgz
+        sha256: b8b1b94651e7a4cfd6335180697cd35dda771058f942900b5775a69720d218c8
+        strip-components: 1
+    buildsystem: simple
+    build-commands:
+      - "install bin/steamlink /app/bin"
+      - "cp -a lib/* /app/lib"
+      - "cp -a *.txt ThirdPartyLegalNotices.* /app"
+
+  - name: packaging
+    sources:
+      - type: file
+        path: com.valvesoftware.SteamLink.appdata.xml
+      - type: file
+        path: com.valvesoftware.SteamLink.desktop
+      - type: dir
+        path: icons
+    buildsystem: simple
+    build-commands:
+      - install -d /app/share/applications/
+      - install -d /app/share/metainfo/
+      - install -D -m644 com.valvesoftware.SteamLink.appdata.xml /app/share/metainfo/
+      - install -D -m644 com.valvesoftware.SteamLink.desktop /app/share/applications/
+      - |
+        set -eu; for size in 16 24 32 48 256; do
```

Comment:
I think it's fine to stick to 128x128 and 256x256.

Reply:
These are the icons I got from Valve. I can drop the smaller versions if you want, but I'll have to ask for a 128px version (if the build doesn't automatically scale it down from 256).

Reply:
It should scale down the biggest one so it's fine.

Reply:
The build doesn't seem to do any scaling automatically, so the desktop environment will be responsible for scaling the 256px icon down to its preferred size. I assume that's OK? This seems to be the same thing that will happen for `com.valvesoftware.Steam`.

If that's a problem, I can either mechanically scale it down to 128px during build, or go back to Valve UI people and ask for a "perfect" 128px icon (or a SVG, but I don't know whether the source material for the icon exists as a SVG).

Reply:
It's nice to have once it moves to stable, but I guess at least GNOME just picks up the biggest available icon or SVG, so I don't consider it a blocker.